README.md

```diff
@@ -11,6 +10,12 @@ The HTTP listener redirects to HTTPS.
 
 The HTTPS listener uses a certificate stored in ACM or IAM.
 
+## Terraform Versions
+
+Terraform 0.12. Pin module version to ~> 2.0. Submit pull-requests to master branch.
+
+Terraform 0.11. Pin module version to ~> 1.0. Submit pull-requests to terraform011 branch.
```

Comment:
This module is already on v2.6.3, so it may need to be 

0.12 ~>  3.0
0.11 ~> 2.0